Tests/AnnoDesigner.Tests/Undo/UndoManagerTests.cs

```diff
@@ -4,218 +4,260 @@
 using Moq;
 using Xunit;
 
-namespace AnnoDesigner.Tests
+namespace AnnoDesigner.Tests.Undo
 {
     public class UndoManagerTests
     {
-        public UndoManager UndoManager => new UndoManager();
+        private UndoManager _undoManager;
 
-        private void DoNothing()
+        public UndoManagerTests()
         {
-
+            _undoManager = new UndoManager();
```

Comment:
Tests will be at least partially dependant on each other. Better move this to Setup, or revert back to getter which always create new instance.

Reply:
In xUnit.net the ctor of a test class is run before every single test in that class.
So it will always be a different instance -> no interference with other tests.
[See for clarification](https://xunit.net/docs/shared-context#constructor)